Makefile

```diff
@@ -57,7 +57,13 @@ clean:
 	rm -f dist/flannel*
 	rm -f release/cni-plugin-flannel*
 
-release: vendor
+package:
+	scripts/package.sh
+
+release: build_all
+	scripts/package.sh
```

Comment:
Shouldn't this call `scripts/release.sh`?

Reply:
originally, I was going to take that path. However, the issue it presented was that a user had no method of testing the package process without running a complete build every time. Switching to using package.sh allows a user to build all of the binaries and tarfiles once and then run the package script multiple times without needing to re-do the compute portion of the builds. 
This also ties into the intention of release.sh being more of a dev/debug/qa process as it builds everything in a docker container and then copies the binaries/tarfiles onto the host. In theory, it could also be a "legacy" method of publishing a manual release.

This change also allows us to hopefully use the github actions cache action by separating the two steps.

Reply:
ok, thanks for the explanation